Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -62,6 +61,9 @@ bool NANDImporter::ReadNANDBin(const std::string& path_to_bin)
 
   for (size_t i = 0; i < NAND_TOTAL_BLOCKS; i++)
   {
+    if (i % 1000 == 0)
```

Comment:
Oh, commented at the same time: if you do want to limit the number of reports; at least leave a comment that it is intentional and mostly just a-decent-number-that-works™. Same applies if you decide to change it back to the old code with progress reporting (but don't forget to divide the count by that number if you send scaled numbers)